README.md

```diff
@@ -6,195 +6,9 @@ Language for building DeFi apps on Radix.
-| Reset ledger state                 | `resim reset`                                                              |
-
-**Note:** The commands use the default account as transaction sender.
-
-## Compile blueprints with dockerized `radix-clis`
-
-Follow this guide to build reproducible WASM and RDP files for your Scrypto blueprints.
-
-### Using local docker image
-
-The Dockerfile in the root of the repo should be work to build a docker image which will contain all the dependencies needed to be able build a blueprint using scrypto.
-
-Build the docker image like. From the repo root
-
-```bash
-docker build -t radixdlt/radix-clis .
-```
-
-Then build your package by just running
-
-```bash
-docker run -v <path-to-your-scrypto-crate>:/src radixdlt/radix-clis
-```
-
-### Using published docker image
-
-If you would like to avoid building the docker image, you can skip the build step and do the second step directly, docker will automatically download the docker image we publish
-
-Build your blueprints directly with
-
-```bash
-docker run -v <path-to-your-scrypto-crate>:/src radixdlt/radix-clis
-```
-
-## Project Layout
-
-- `radix-blueprint-schema-init`: Blueprint schema initialization structures, used by Radix Package Definition (RPD).
-- `radix-clis`: Various CLI tools, like `resim`, `scrypto`, `rtmc` and `rtmd`.
-- `radix-common-derive`: Macros for defining `Decimal` and `PreciseDecimal`.
-- `radix-common`: Common libraries used by Radix Engine and Scrypto.
-- `radix-engine`: The Radix Engine implementation.
-- `radix-native-sdk`: Library to assist native blueprint development.
-- `radix-sbor-derives`: Macros for encoding and decoding Scrypto SBOR and Manifest SBOR data.
-- `radix-substate-store-impls`: Various substate store implementations.
-- `radix-substate-store-interface`: The interface of any substate store.
-- `radix-substate-store-queries`: Interprets data in substate data by injecting high-level knowledge.
-- `radix-transaction-scenarios`: Defines various transaction scenarios, for testing.
-- `radix-transactions`: Radix transaction manifest compiler, transaction models, signing and validating logic.
-- `sbor-derive`: Macros for encoding and decoding SBOR data.
-- `sbor`: A generic binary data format, upon which Scrypto SBOR and Manifest SBOR are built.
-- `scrypto-derive`: Macros for defining blueprints.
-- `scrypto-test`: Library for testing Scrypto blueprints.
-- `scrypto`: Scrypto language abstraction.
-
-## LFS
-
-Assets under `assets-lfs` are stored in Git LFS.
-
-To fetch files from LFS, install `git-lfs` first:
-
-- MacOS
-  ```
-  brew install git-lfs
-  ```
-- Ubuntu
-  ```
-  curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | sudo bash
-  sudo apt-get install git-lfs
-  ```
-
-and then:
-
-```
-git lfs install
-git lfs pull
-```
+To set up your development environment, please following the instructions on https://docs.radixdlt.com/docs/getting-rust-scrypto.
```

Comment:
Looks good and approving, although I think we miss this bit:
* The docs in action.yaml need updating to tell developers to update the docs instead of updating the readme